lib/Makefile

```diff
@@ -305,13 +305,21 @@ else
   PCLIB := $(LDFLAGS_DYNLIB)
 endif
 
-ifneq (,$(filter $(UNAME),FreeBSD NetBSD DragonFly))
+
+ifneq ($(MT),)
```

Comment:
That section is a bit unclear to me.
Why was it changed ?
It seems it just duplicates the section above.